### ISSUE TITLE
UI edge case to handle

### ISSUE THREAD

**Issue by skrawcz** [DAGWorks-Inc/burr#72]
Handle multiple entries for the same sequence ID.

Use case -- tracker logged an error. I then restart from that state and fix the bug, but I want to continue to trace to the same place. This will result in something like this:

```javascript
{"type":"begin_entry","start_time":"2024-03-14T23:46:33.550378","action":"counter","inputs":{},"sequence_id":2}
{"type":"end_entry","end_time":"2024-03-14T23:46:33.551172","action":"counter","result":null,"exception":"Traceback (most recent call last):\n  File \"/Users/stefankrawczyk/dagworks/burr/burr/core/application.py\", line 320, in _step\n    raise e\n  File \"/Users/stefankrawczyk/dagworks/burr/burr/core/application.py\", line 310, in _step\n    result, new_state = _run_single_step_action(next_action, self._state, inputs)\n  File \"/Users/stefankrawczyk/dagworks/burr/burr/core/application.py\", line 186, in _run_single_step_action\n    result, new_state = action.run_and_update(state, **inputs)\n  File \"/Users/stefankrawczyk/dagworks/burr/burr/core/action.py\", line 442, in run_and_update\n    return self._fn(state, **self._bound_params, **run_kwargs)\n  File \"/Users/stefankrawczyk/dagworks/burr/examples/counter/application.py\", line 18, in counter\n    raise ValueError(\"random error\")\nValueError: random error\n","state":{"counter":1,"__SEQUENCE_ID":2,"__PRIOR_STEP":"counter"},"sequence_id":2}
{"type":"begin_entry","start_time":"2024-03-14T23:55:09.384311","action":"counter","inputs":{},"sequence_id":2}
{"type":"end_entry","end_time":"2024-03-14T23:55:09.385772","action":"counter","result":{"counter":2},"exception":null,"state":{"counter":2,"__SEQUENCE_ID":2,"__PRIOR_STEP":"counter"},"sequence_id":2}
```
So there will be two entries. The UI just shows the last one. Should really show both...

Alternatively, we should never write to the same place and deal with this problem differently.

**Comment by elijahbenizzy** [comment]
Fixed by #73

**Comment by elijahbenizzy** [comment]
This is now complete I think...
